core/layout_api.php

```diff
@@ -561,13 +561,15 @@ function layout_navbar_button_bar() {
 	if( $t_show_report_bug_button )  {
 		$t_bug_url = string_get_bug_report_url();
 		echo '<a class="btn btn-primary btn-sm" href="' . $t_bug_url . '">';
-		echo '<i class="fa fa-edit"></i> ' . lang_get( 'report_bug_link' );
+		print_icon( 'fa-edit');
+		echo lang_get( 'report_bug_link' );
```

Comment:
Should be `echo ' ' . lang_get( 'report_bug_link' );` to have the same look like before
Now
![grafik](https://user-images.githubusercontent.com/319317/103461986-73aee280-4d22-11eb-9d4e-1debda12b51f.png)
Before
![grafik](https://user-images.githubusercontent.com/319317/103462003-993bec00-4d22-11eb-8bc9-477c337955e8.png)

Reply:
Good catch